app/client/src/components/shared/MenuList.tsx

```diff
@@ -36,44 +22,13 @@ export function MenuList<T>(props: MenuListProps<T>) {
   return (
     <>
       {hasSubChildren && children[0].props.label}
-      <VariableSizeList
-        ref={listRef}
-        width="100%"
-        height={maxHeight}
-        itemCount={childrenToDisplay.length}
-        itemSize={getSize}
-      >
-        {({ index, style }) => (
-          <div style={{ ...style, overflowX: 'hidden' }}>
-            <MenuItem
-              index={index}
-              width={width}
-              setSize={setSize}
-              value={childrenToDisplay[index]}
-            />
-          </div>
-        )}
-      </VariableSizeList>
+      <Virtuoso
+        itemContent={(index) => <div>{childrenToDisplay[index]}</div>}
```

Comment:
I'm seeing some performance issues with this. 

The `Waterbody Names` menu is very slow to scroll on the http://localhost:3000/state/PA/advanced-search page. There are 216,159 waterbodies in that list.

Reply:
Good call, the menu isn't even opening for me, I will look into this.